packages/core-orders/src/module/configureOrdersModule-queries.ts

```diff
@@ -22,6 +28,24 @@ export const buildFindSelector = ({ includeCarts, status, userId, queryString }:
   return selector;
 };
 
+const normalizeOrderAggregateResult = (data = []): OrderReport => {
+  const statusToFieldMap = {
```

Comment:
No, you should not map the status to count:

An order that has been created yesterday and has been fullfilled today should lead to createdCount +1 when the report does not cover today and fullfillfedCount +1 when the report includes today.

Reply:
Use the dates instead, there is:

  created: Date;
  confirmed?: Date;
  rejected?: Date;
  fullfilled?: Date;